extensions/functions_arithmetic.yaml

```diff
@@ -276,3 +276,105 @@ aggregate_functions:
         decomposable: MANY
         intermediate: "STRUCT<fp64,i64>"
         return: fp64?
+  - name: "min"
+    description: Min a set of values.
+    impls:
+      - args:
+          - options: [ SILENT, SATURATE, ERROR ]
+            required: false
+          - value: i8
+        nullability: DECLARED_OUTPUT
+        decomposable: MANY
+        intermediate: i8
```

Comment:
Seems both the `intermediate` and `return` type should be `i8?`, indicating that a nullable i8.  

Same applied to other impls.

Reply:
Got it. BTW do we need to define another function definication that the `return` type is not nullable ?

Reply:
Got it. BTW do we need to define another function definication that the `return` type is not nullable ? The [specification](https://substrait.io/extensions/) shows that the functoin signature (which is the key in the yaml file) doesn't include the return type.

Reply:
done